warehouse/integrations/vulnerabilities/osv/__init__.py

```diff
@@ -0,0 +1,108 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import time
+
+import requests
+
+from warehouse import integrations
+from warehouse.integrations import vulnerabilities
+
+
+class OSVPublicKeyAPIError(vulnerabilities.InvalidVulnerabilityReportRequest):
+    pass
+
+
+OSV_PUBLIC_KEYS_URL = "https://osv.dev/public_keys/pypa"
```

Comment:
As of today, this URL isn't set up yet. You may want to make sure it's all good before merging (or at least before sending payloads our way)

Reply:
I think it may be a good idea to set this as a envvar defined app setting. I had to do that with GitHub too, this made testing much much easier

Reply:
This is still an empty/useless page which blocks merge.

Reply:
endpoint is up